.travis.yml

```diff
@@ -6,23 +6,19 @@ go:
 
 env:
   matrix:
-  - DISTRIBUTION=ubuntu
-  - DISTRIBUTION=debian
-  - DISTRIBUTION=centos
-  - DISTRIBUTION=rhel
-  - DISTRIBUTION=opensuse
-  - DISTRIBUTION=sles
+  - PACKAGE_MANAGER=dpkg
+  - PACKAGE_MANAGER=rpm
 
-script: make test/$DISTRIBUTION
+script: make test
 
 before_install:
   - sudo apt-get -q update
   - sudo apt-get install -y make gcc rpm
 
 before_deploy:
   - gem install fpm --no-document
-  - VERSION=${TRAVIS_TAG:=$(cat ./VERSION)} make pkg/$DISTRIBUTION
-  - sed -i'' -e "s/\DISTRIBUTION/${DISTRIBUTION}/" descriptor.json
+  - VERSION=${TRAVIS_TAG:=$(cat ./VERSION)} make pkg/$PACKAGE_MANAGER
+  - sed -i'' -e "s/\PACKAGE_MANAGER/${PACKAGE_MANAGER}/" descriptor.json
```

Comment:
1. Set version `env` to current version and build packages. 
2. replace PACKAGE_MANAGER with package manager from matrix in `descriptor.json` file.
I forgot to change `DISTRIBUTION` in descriptor.json file with `PACKAGE_MANAGER` or maybe I should ask before I changed those lines because I don't know what consequence this will have for deploying process.